config/settings.yml

```diff
@@ -137,6 +137,10 @@ sys_api_key:
   default: ''
 repository_authentication_caching_enabled:
   default: 1
+repositories_automatic_managed_repos:
+  default: 0
```

Comment:
Ah, sure. I wasn't sure whether bool was working for the current functionality, as no other setting seemed to use it.